src/rtp/ntp.c

```diff
@@ -64,15 +79,26 @@ int ntp_time_get(struct ntp_time *ntp)
 	GetSystemTimeAsFileTime(&now.ft);
 	tv.tv_usec = (long) ((now.ns100 / 10LL) % 1000000LL);
 	tv.tv_sec = (long) ((now.ns100 - 116444736000000000LL) / 10000000LL);
+
+	*jfs = now.ns100 / (10 * 1000);
 #else
 	if (gettimeofday(&tv, NULL) != 0)
 		return errno;
+
+	*jfs  = (uint64_t)tv.tv_sec * (uint64_t)1000;
+	*jfs += tv.tv_usec / 1000;
 #endif
 	unix2ntp(ntp, &tv);
 
 	return 0;
 }
 
+int ntp_time_get(struct ntp_time *ntp)
+{
+	uint64_t jfs;
+	return get_ntp_time_jiffies(ntp, &jfs);
```

Comment:
can you make jfs optional, so that we can pass a NULL pointer in case we dont need it ?

Reply:
Yes, I can do it.